website/client/components/tasks/taskModal.vue

```diff
@@ -249,6 +251,7 @@
 
 <style lang="scss">
   @import '~client/assets/scss/colors.scss';
+  @import '~client/assets/scss/datepicker.scss';
```

Comment:
Hmm this is going to work only in the tasks page, instead of importing this one here can you add the new css file to `website/client/assets/scss/index.scss`? thanks!